Gemfile

```diff
@@ -36,6 +36,12 @@ gem "sentry-rails"
 gem "rails-settings-cached"
 gem "octokit"
 
+# Active Storage
+gem 'aws-sdk-s3', '~> 1'
+gem 'azure-storage-blob', '~> 1.0', '>= 1.0.1'
```

Comment:
Is there a specific reason for using a different versions from the Rails guides for AWS and Azure?

```
gem 'aws-sdk-s3', require: false
gem 'azure-storage-blob', "~> 2.0", require: false
```

Reply:
Nope. I missed that in the guide.